.github/workflows/ci.yml

```
@@ -15,15 +15,15 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - pair:
-              elixir: 1.10.4
-              otp: 22.3.4
           - pair:
               elixir: 1.11.4
               otp: 23.2.7
           - pair:
               elixir: 1.13.3
               otp: 24.2.2
+          - pair:
+              elixir: 1.14.5
+              otp: 25.3.2
```

I think we should go with the latest ones here.

```suggestion
          - pair:
              elixir: 1.15.0
              otp: 26.0.1
```